src/psm/test/corners.ok

```diff
@@ -4,29 +4,25 @@
 [INFO ODB-0131]     Created 624 components and 2752 component-terminals.
 [INFO ODB-0132]     Created 2 special nets and 1248 connections.
 [INFO ODB-0133]     Created 581 nets and 1504 connections.
-[INFO PSM-0001] Reading voltage source file: Vsrc_gcd_vdd.loc.
-[INFO PSM-0015] Reading location of VDD and VSS sources from Vsrc_gcd_vdd.loc.
-[INFO PSM-0076] Setting metal node density to be standard cell height times 5.
-[WARNING PSM-0030] VSRC location at (50.000um, 50.000um) and size 20.000um, is not located on an existing power stripe node. Moving to closest node at (68.070um, 53.115um).
-[INFO PSM-0031] Number of PDN nodes on net VDD = 604.
-[INFO PSM-0064] Number of voltage sources = 1.
-[INFO PSM-0040] All PDN stripes on net VDD are connected.
+[INFO PSM-0040] All shapes on net VDD are connected.
+[INFO PSM-0015] Reading location of sources from: Vsrc_gcd_vdd.loc.
 ########## IR report #################
-Corner: min
-Worstcase voltage: 5.76e-01 V
-Average IR drop  : 4.75e-01 V
-Worstcase IR drop: 5.24e-01 V
+Corner           : min
+Supply voltage   : 9.50e-01 V
+Worstcase voltage: 9.12e-01 V
+Average voltage  : 9.25e-01 V
+Average IR drop  : 2.51e-02 V
+Worstcase IR drop: 3.82e-02 V
+Percentage drop  : 4.02 %
 ######################################
-[INFO PSM-0001] Reading voltage source file: Vsrc_gcd_vdd.loc.
-[INFO PSM-0015] Reading location of VDD and VSS sources from Vsrc_gcd_vdd.loc.
-[INFO PSM-0076] Setting metal node density to be standard cell height times 5.
-[WARNING PSM-0030] VSRC location at (50.000um, 50.000um) and size 20.000um, is not located on an existing power stripe node. Moving to closest node at (68.070um, 53.115um).
-[INFO PSM-0031] Number of PDN nodes on net VDD = 604.
-[INFO PSM-0064] Number of voltage sources = 1.
-[INFO PSM-0040] All PDN stripes on net VDD are connected.
+[INFO PSM-0040] All shapes on net VDD are connected.
+[INFO PSM-0015] Reading location of sources from: Vsrc_gcd_vdd.loc.
 ########## IR report #################
-Corner: max
-Worstcase voltage: 6.25e-01 V
-Average IR drop  : 4.30e-01 V
-Worstcase IR drop: 4.75e-01 V
+Corner           : max
+Supply voltage   : 9.50e-01 V
+Worstcase voltage: 9.30e-01 V
+Average voltage  : 9.41e-01 V
+Average IR drop  : 8.54e-03 V
+Worstcase IR drop: 2.02e-02 V
```

Comment:
That's a fairly large change in results - what do you attribute it to?

Reply:
There is one bug in there, which I can fix. There is another source of differences which relate to how the old and new implementations handle parallel vias (the old one just made it 1 via, while mine will merge the vias and reduce the resistance of that connection).

Reply:
If I triple the resistance of the vias (the emulate only using a single via instead instead of 3 parallel vias) I get:
```
########## IR report #################
Net              : VDD
Corner           : min
Supply voltage   : 1.10e+00 V
Worstcase voltage: 5.37e-01 V
Average voltage  : 5.76e-01 V
Average IR drop  : 5.24e-01 V
Worstcase IR drop: 5.63e-01 V
Percentage drop  : 51.19 %
######################################
```

Compared with the solution on master:
```
########## IR report #################
Corner: min
Worstcase voltage: 5.76e-01 V
Average IR drop  : 4.75e-01 V
Worstcase IR drop: 5.24e-01 V
######################################
```

This is much closer to the original solution.